user-service/src/main/java/rs/edu/raf/banka1/bootstrap/BootstrapData.java

```diff
@@ -72,15 +72,19 @@ public void run(String... args) throws Exception {
         client.setPassword(passwordEncoder.encode("client"));
         client.setFirstName("Client");
         client.setActive(true);
-        user1.setPosition("employee");
+        client.setPosition("employee");
```

Comment:
Ovde ćeš potencijalno imati konfikta sa Vidom.

Reply:
dap, ali vidove izmene nisu jos na sprint-4 grani, tako da cu to kasnije kad se merguje njegov kod